### DISCUSSION TITLE
Firestick remote

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#340] [General]
I lost my firestick remote. Any ideas if I can buy just a remote without purchasing a new FireStick ?

**Comment by Bibett** [comment]
I guess it's a good idea to look for remote on the internet.

**Comment by hernessteve** [comment]
You can easily use your phone as a remote. There are few other options available in this article - [https://www.firesticktricks.com/pair-firestick-remote.html](url). Firestick tricks has tons of useful stuff about firetick and different settings/addons etc. Hope it helps
